sources/web/datalab/static.ts

```diff
@@ -159,14 +160,51 @@ function sendUserCustomTheme(userId: string, response: http.ServerResponse): voi
 }
 
 /**
+ * Returns true if this path should return an experimental UI resource
+ * @param path the incoming request path
+ */
+export function isExperimentalResource(pathname: string) {
+  const experimentalUiEnabled = process.env.DATALAB_EXPERIMENTAL_UI;
```

Comment:
this is the only place, I think I'd like to keep it this way since it's meant as temporary code to be removed, I don't want to flesh it out too much.